src/Files.App/Helpers/ShellContextMenuHelper.cs

```diff
@@ -199,7 +198,14 @@ async void InvokeShellMenuItem(ContextMenu contextMenu, object? tag)
 						await contextMenu.InvokeItem(menuId);
 						break;
 				}
+				
+				void InstallFont(string path, bool asAdmin)
+				{
+					string dir = asAdmin ? Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Windows), "Fonts")
+						: Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "Microsoft", "Windows", "Fonts");
 
+					Win32API.RunPowershellCommand($"-command \"Copy-Item '{path}' '{dir}'; New-ItemProperty -Name '{Path.GetFileNameWithoutExtension(path)}' -Path 'HKCU:\\Software\\Microsoft\\Windows NT\\CurrentVersion\\Fonts' -PropertyType string -Value '{dir}'\"", asAdmin);
```

Comment:
When installing as admin you should use HKLM not HKCU.

Reply:
Seems we'll need another var for registry path.

Reply:
Done